docs/concepts/artifacts.md

```diff
@@ -9,36 +9,43 @@ search:
-- Documentation: Artifacts can be used to publish documentation and sample data to help you keep track of your work and share information with your colleagues. For instance, artifacts allow you to add a description to let your colleagues know why this piece of data is important. 
+- Documentation: Artifacts can be used to publish documentation and sample data to help you keep track of your work and share information with your colleagues. For instance, artifacts allow you to add a description to let your colleagues know why this piece of data is important.
 
-## Creating Artifacts
+## Creating artifacts
 
-Creating artifacts allows you to publish data from task and flow runs or outside of a flow run context. Currently, you can render three artifact types: links, markdown, and tables.
+Creating artifacts allows you to publish data from task and flow runs or outside of a flow run context.
+Currently, you can render three artifact types: links, Markdown, and tables.
 
 !!! note "Artifacts render individually"
-    Please note that every artifact created within a task will be displayed as an individual artifact in the Prefect UI. This means that each call to `create_link_artifact()` or `create_markdown_artifact()` generates a distinct artifact. 
-    
+    Please note that every artifact created within a task will be displayed as an individual artifact in the Prefect UI.
+    This means that each call to `create_link_artifact()` or `create_markdown_artifact()` generates a distinct artifact.
+
     Unlike the `print()` command, where you can concatenate multiple calls to include additional items in a report, within a task, these commands must be used multiple times if necessary. 
     
     To create artifacts like reports or summaries using `create_markdown_artifact()`, compile your message string separately and then pass it to `create_markdown_artifact()` to create the complete artifact.
 
-### Creating Link Artifacts
-
-To create a link artifact, use the `create_link_artifact()` function. To create multiple versions of the same artifact and/or view them on the Artifacts page of the Prefect UI, provide a `key` argument to the `create_link_artifact()` function to track an artifact's history over time. Without a `key`, the artifact will only be visible in the artifacts tab of the associated flow run or task run."
-
+### Creating link artifacts
 
+To create a link artifact, use the `create_link_artifact()` function.
+To create multiple versions of the same artifact and/or view them on the Artifacts page of the Prefect UI, provide a `key` argument to the `create_link_artifact()` function to track an artifact's history over time.
+Without a `key`, the artifact will only be visible in the artifacts tab of the associated flow run or task run."
```

Comment:
```suggestion
Without a `key`, the artifact will only be visible in the Artifacts tab of the associated flow run or task run."
```
I believe our Prefect UI pages should be capitalized but I can't recall if that extends to tabs.

Reply:
I think you're right. I'll check. If it's an element on a page the convention (and now our style guide) says to bold it.

Reply:
🤔

Reply:
The style guide doesn't specify, but ChatGPT says that's the standard, so added to the style guide.